src/h5web/providers/Provider.tsx

```diff
@@ -15,13 +15,18 @@ function Provider(props: Props): ReactElement {
 
   // Wait until metadata is fetched before rendering app
   const { value: metadata, error } = useAsync(async () => {
-    return api.fetchMetadata();
+    return api.getMetadata();
   }, [api]);
 
-  const valuesStore = useMemo(
-    () => createFetchStore(api.fetchValue.bind(api)),
-    [api]
-  );
+  const groupsStore = useMemo(() => {
+    const store = createFetchStore(api.getGroup.bind(api));
+    store.prefetch('/'); // pre-fetch root group
+    return store;
+  }, [api]);
```

Comment:
The new `groupsStore` that wraps the `getGroup` method. Here we also prefetch the root group to shave off a few millisecond as per React's "fetch as you render" pattern.